readme.md

```diff
@@ -162,13 +220,13 @@ Your Rails app has the following `application.html.erb`. Nothing shows up in you
 </html>
 ```
 
-> Your answer...
+> It's that darn title tag again, it needs a closing ">"
```

Comment:
This is an error, but it doesn't break the html.  Do you see another error that will make it so each template does not show in the layout?